src/boot/efi/linux-aarch64.c

```diff
@@ -0,0 +1,161 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <efi.h>
+#include <efilib.h>
+#include <libfdt.h>
+
+#include "linux.h"
+#include "linux-aarch64.h"
+
+/* DTB table GUID, as defined by UEFI specification 2.9 */
+/* gnu-efi after 3.0.13 should already define this */
+#ifndef EFI_DTB_TABLE_GUID
+#define EFI_DTB_TABLE_GUID \
+    { 0xb1b621d5, 0xf19c, 0x41a5, {0x83, 0x0b, 0xd9, 0x15, 0x2c, 0x69, 0xaa, 0xe0} }
+static EFI_GUID EfiDtbTableGuid = EFI_DTB_TABLE_GUID;
```

Comment:
this should be `const`, no?

That said, we nowadays would define the following:

```c
#define EFI_DTB_TABLE_GUID \
        &(const EFI_GUID) { 0xb1b621d5, 0xf19c, 0x41a5, {0x83, 0x0b, 0xd9, 0x15, 0x2c, 0x69, 0xaa, 0xe0} }
```

And then use that directly, without defining a static variable first, ie. use C99 compound literals